src/i18n/en-us/index.js

```diff
@@ -130,7 +133,6 @@ export default {
         paste_contract_code_here: 'copy & paste contract code here...',
         enter_contract_text: 'enter or paste contract text',
         verify_contract: 'Verify Contract',
-        dismiss: 'Dismiss',
```

Comment:
moved 'dismiss' to namespace `global`

Reply:
Ok. This will require replicating the same index.js refactor in the incoming Portuguese PR.
I'll take care of it